src/track/albuminfo.h

```diff
@@ -33,9 +33,6 @@ class AlbumInfo final {
     AlbumInfo& operator=(AlbumInfo&&) = default;
     AlbumInfo& operator=(const AlbumInfo&) = default;
 
-    // TODO(XXX): Remove after all new fields have been added to the library
-    void resetUnsupportedValues();
```

Comment:
I do not understand the original code and the change. Can you give a hint?

Reply:
The original implementation relied on null values as a marker. Fields with null values were not exported to prevent overwriting valid file tags with empty, missing fields.